src/main/java/seedu/address/model/person/ClientPerson.java

```diff
@@ -37,15 +37,6 @@ public void setId(Id updatedId) {
         this.clientId = updatedId;
     }
 
-    /**
-     * Returns true if both persons of the same name have at least one other identity field that is the same.
-     * This defines a weaker notion of equality between two persons.
-     */
```

Comment:
What does client person do? I tot only got bidder and seller and person or is this diannes thing for property